### PR TITLE
add mainnet fork epoch

### PR DESCRIPTION
**Pull request by potuz**
Blobs are coming. 

https://github.com/ethereum/consensus-specs/pull/3597

### CODE REVIEW

File: config/params/mainnet_config.go [hunk 1]
```diff
@@ -23,7 +23,10 @@ const (
 	mainnetAltairForkEpoch = 74240 // Oct 27, 2021, 10:56:23am UTC
 	// Bellatrix Fork Epoch for mainnet config.
 	mainnetBellatrixForkEpoch = 144896 // Sept 6, 2022, 11:34:47am UTC
-	mainnetDenebForkEpoch     = math.MaxUint64
+	// Capella Fork Epoch for mainnet config.
+	mainnetCapellaForkEpoch = 194048 // April 12, 2023, 22:27:35 UTC
```
Comment (potuz) [inline config/params/mainnet_config.go:27]: For consistency... don't complain about 22 instead of 10pm.

Comment (prestonvanloon) [reply config/params/mainnet_config.go:27]: 22 is the way

**Comment by prestonvanloon** [comment]
Tests are failing since we use the consensus_specs repo to verify our configs are correct and we've only updated eth-networks

**Comment by prestonvanloon** [comment]
Waiting for https://github.com/ethereum/consensus-specs/pull/3598

**Review by prestonvanloon — APPROVED** [review]

File: config/params/mainnet_config.go (hunk 1 above)
Comment (saolyn) [inline config/params/mainnet_config.go:27]: That being said, for consistency or at least appearing the same as the above comments, it should be: 
```suggestion
	mainnetCapellaForkEpoch = 194048 // April 12, 2023, 22:27:35pm UTC
```

File: config/params/mainnet_config.go [hunk 2]
```diff
@@ -23,7 +23,10 @@ const (
...
+	// Deneb Fork Epoch for mainnet config.
+	mainnetDenebForkEpoch = 269568 // March 13, 2024, 13:55:35 UTC
```
Comment (saolyn) [inline config/params/mainnet_config.go:29]: Same here:
```suggestion
	mainnetDenebForkEpoch = 269568 // March 13, 2024, 13:55:35pm UTC
```

**Review by prestonvanloon — APPROVED** [review]

**Review by terencechain — APPROVED** [review]

**Pull request merged**